cypress/integration/devices_page_test.js

```diff
@@ -50,5 +50,18 @@ describe('Devices page tests', () => {
       });
       cy.location('pathname').should('eq', '/devices/register');
     });
+
+    it('correctly filters by connection status', () => {
+      cy.get('#checkbox-connected').check();
+      cy.get('#checkbox-disconnected').uncheck();
+      cy.get('#checkbox-never-connected').uncheck();
+      cy.get('table tbody tr i').should('have.class', 'icon-connected');
+    });
+
+    it('correctly filters by device handle', () => {
+      const filter = "_dev";
+      cy.get('#filterId').type(filter);
+      cy.get('table tbody tr').contains(filter);
+    });
```

Comment:
Do you think we should add tests to check if pagination works correctly?

Reply:
We would need fixtures with lot of devices. I'd switch to factories before adding that test